block/src/Pos/Block/Logic/VAR.hs

```diff
@@ -98,13 +108,34 @@ verifyBlocksPrefix blocks = runExceptT $ do
 type BlockLrcMode ctx m = (MonadBlockApply ctx m, LrcModeFull ctx m)
 
 -- | Applies blocks if they're valid. Takes one boolean flag
--- "rollback". Returns header hash of last applied block (new tip) on
--- success. Failure behaviour depends on "rollback" flag. If it's on,
--- all blocks applied inside this function will be rollbacked, so it
+-- @rollback@. Returns header hash of last applied block (new tip) on
+-- success. Failure behaviour depends on the @rollback@ flag. If it's on,
+-- all blocks applied inside this function will be rolled back, so it
 -- will do effectively nothing and return 'Left error'. If it's off,
--- it will try to apply as much blocks as it's possible and return
--- header hash of new tip. It's up to caller to log warning that
--- partial application happened.
+-- it will try to apply as many blocks as it's possible and return
+-- header hash of new tip. It's up to the caller to log a warning that
+-- partial application has occurred.
+--
+-- Algorithm:
```

Comment:
It's way too descriptive.

Reply:
I disagree because this is a thorough description of the code that spans multiple files and well over a hundred LOC, and the description itself is below 20 lines. Do you have any concrete suggestions on how to make it more concise?

Reply:
> well over a hundred LOC

Nobody needs to know implementation details ;)

>Do you have any concrete suggestions on how to make it more concise?

I believe you can do it yourself. Try using shorter words, rephrase something.